### DISCUSSION TITLE
Get pathology information in lung cancer data

### DISCUSSION THREAD

**Discussion by mareenajms** [synthetichealth/synthea#1390] [Q&A]
Hi team,
I am new to Synthea data generator tool. I am running a default .jar file in basic setup for generating data in my project. By passing lung_cancer as module parameter in terminal, I am able to generate lung cancer patient data. But none of the generated data has pathology information. Am I missing something during generating ? Is there any way I can get pathology information also in generated data?

Thanks

**Comment by jawalonoski** [comment]
Are you using `-m` parameter? Please see this https://github.com/synthetichealth/synthea/wiki/The--M-Feature

If you only want patients with lung cancer, trying to create "Keep Module" instead. See https://github.com/synthetichealth/synthea/wiki/Keep-Patients-Module
